templates/rfc_template.md

```diff
@@ -0,0 +1,65 @@
+# Title of My Design
+
+
+## [Optional] Subtitle of my design 
+
+**Authors:** [Nice Person](mailto:some.nice.person@chanzuckerberg.com)
+
+**Approvers:** [Friendly Tech Person](mailto:some.nice.person@chanzuckerberg.com), [Friendly PM Person](mailto:some.nice.person@chanzuckerberg.com), [Friendly Tech Person2](mailto:some.nice.person@chanzuckerberg.com)
+
```

Comment:
@maniarathi Can we add the **Consulted** section with IE (+ cc @patangay) back? I think it's fine to call out if the RFC isn't technical in nature this section can be skipped.

Reply:
Done!